src/Groups/group_constructors.jl

```diff
@@ -139,23 +161,28 @@ end
 ################################################################################
 
 """
-    free_group
+    free_group(n::Int, s::String = "f") -> FPGroup
+    free_group(L::Vector{String}) -> FPGroup
+    free_group(L::String...) -> FPGroup
+
+The first form returns the free group of rank `n`, where the generators are
+printed as `s1`, `s2`, ..., the default being `f1`, `f2`, ...
+
+The second form, if `L` has length `n`, returns the free group of rank `n`,
+where the `i`-th generator is printed as `L[i]`.
 
-There are four ways to define a free group.
-- `free_group(n::Int) -> FPGroup`; return the free group of rank `n`, with generators printed as `"f1"`,`"f2"`,`"f3"`, etc.
-- `free_group(L::String...) -> FPGroup`; return the free group with length(`L`) generators, printed as `L[1]`, `L[2]`, `L[3]`, etc.
-- `free_group(L::Array{String,1}) -> FPGroup`; same as above.
-- `free_group(n::Int, s::String) -> FPGroup`; return the free group of rank `n`, with generators printed as `"s1"`, `"s2"`, `"s3"`, etc.
+The second form, if there are `n` arguments `L...`,
```

Comment:
Maybe this was meant?
```suggestion
The third form, if there are `n` arguments `L...`,
```

Reply:
Yes.